### PR TITLE
Rename token module endpoints 

### LINKED ISSUE #1555: Rename token module endpoints
### Description

All the endpoints in the Lisk Service API following the pattern: `/{module}/...` except the following endpoints in the `token` module:

- GET /tokens
- GET /tokens/summary

### Acceptance Criteria

- Rename `GET /tokens` -> `GET /token/balances` (Make corresponding RPC endpoint updates)
- Rename `GET /tokens/summary` -> `GET /token/summary` (Make corresponding RPC endpoint updates)
- Update the endpoints in the integration tests
- Ensure that the swagger docs are up-to-date

### PR DESCRIPTION
**Pull request by vardan10**
### What was the problem?
This PR resolves #1555

### How was it solved?

- [x] - Renamed GET /tokens to GET /token/balances and updated RPC endpoint.
- [x] - Renamed GET /tokens/summary to GET /token/summary and updated RPC endpoint.
- [x] - Update the endpoints in integration tests
- [x] - Update swagger docs
- [x] - Update filenames
- [x] - Update internal endpoints of indexer
- [x] - Update internal functions of indexer
- [x] - Update variable names inside testcase files

### How was it tested?

- [x] Swagger API call
- [x] Running testcases

### CODE REVIEW

File: services/gateway/apis/http-version3/methods/modules/token/tokens.js [hunk 1]
```diff
@@ -20,8 +20,8 @@ const { transformParams, response, getSwaggerDescription } = require('../../../.
 
 module.exports = {
 	version: '2.0',
-	swaggerApiPath: '/tokens',
-	rpcMethod: 'get.tokens',
+	swaggerApiPath: '/token/balances',
```
Comment (nagdahimanshu) [inline services/gateway/apis/http-version3/methods/modules/token/tokens.js:23]: Can we also rename the file to `tokenBalances`

File: tests/integration/api_v3/http/tokens.test.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const {
 
 const baseUrl = config.SERVICE_ENDPOINT;
 const baseUrlV3 = `${baseUrl}/api/v3`;
-const endpoint = `${baseUrlV3}/tokens`;
+const endpoint = `${baseUrlV3}/token`;
```
Comment (sameersubudhi) [inline tests/integration/api_v3/http/tokens.test.js:31]: Also, please rename the file accordingly. And, kindly do not forget to make the corresponding changes in the RPC tests.
```suggestion
const endpoint = `${baseUrlV3}/token/balances`;
```

File: tests/integration/api_v3/http/tokensSummary.test.js
Comment (sameersubudhi) [inline tests/integration/api_v3/http/tokensSummary.test.js:1]: Corresponding update missing in the RPC tests.

File: services/gateway/tests/constants/registerApi.js [hunk 3]
```diff
@@ -171,8 +171,8 @@ const expectedResponseForRegisterRpcApi = {
 				'get.pos.validators': 'indexer.pos.validators',
 				'get.token.account.exists': 'indexer.token.account.exists',
 				'get.token.constants': 'indexer.token.constants',
-				'get.tokens': 'indexer.tokens',
-				'get.tokens.summary': 'indexer.tokens.summary',
+				'get.token.balances': 'indexer.tokens',
```
Comment (sameersubudhi) [inline services/gateway/tests/constants/registerApi.js:174]: Could we also rename the methods within indexer microservice from `tokens` to `token.balances`? Please leave the method names in connector as is.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: tests/integration/api_v3/http/tokensSummary.test.js
Comment (sameersubudhi) [inline tests/integration/api_v3/http/tokensSummary.test.js:1]: Rename file to `tests/integration/api_v3/http/tokenSummary.test.js`.

File: services/gateway/apis/http-version3/methods/modules/token/tokensSummary.js
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokensSummary.js:1]: Rename file to `services/gateway/apis/http-version3/methods/modules/token/tokenSummary.js`.

File: services/gateway/apis/http-version3/methods/modules/token/tokens.js
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokens.js:1]: Rename file to `services/gateway/apis/http-version3/methods/modules/token/tokenBalances.js`.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**